opencti-platform/opencti-graphql/src/modules/deleteOperation/deleteOperation.graphql

```diff
@@ -0,0 +1,55 @@
+# Type
+type DeletedElement
+{
+    id: String
+    source_index: String
+}
+
+type DeleteOperation implements InternalObject & BasicObject {
+    id: ID!
+    entity_type: String!
+    standard_id: String!
+    parent_types: [String!]!
+    # DeleteOperation
+    timestamp: DateTime
+    user_id: String
+    main_entity_type: String
+    main_entity_id: String
+    main_entity_name: String
+    deleted_elements: [DeletedElement]
+}
+
+# Ordering
+enum DeleteOperationOrdering {
+    timestamp
+}
+
+# Relay connections
+type DeleteOperationConnection {
+    pageInfo: PageInfo!
+    edges: [DeleteOperationEdge!]!
+}
+
+type DeleteOperationEdge {
+    cursor: String!
+    node: DeleteOperation!
+}
+
+# Queries
+type Query {
+    deleteOperation(id: String!): DeleteOperation
```

Comment:
we need to secure all graphql queries with @auth(for: [KNOWLEDGE_KNUPDATE_KNDELETE])

Reply:
i understand needing it for mutations, but do we also need it for queries in our case?

Reply:
I think so, why should we display delete operations if we can't delete objects (or restore them) ? the query need to have the same capability as the one we use to display the bin (to be discussed)